### PR TITLE
Edit *optimade.science* description

### PR DESCRIPTION
**Pull request by blokhin**

### CODE REVIEW

File: clients.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The filtering can be done either using the OPTIMADE filter language (see the spe
 
 ## optimade.science
 
-An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from server, USB-stick, etc.
-It allows for searching through *multiple* OPTIMADE databases using the OPTIMADE filter language (see the specification for more information).
+An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from the server, USB-stick, etc.
+It allows for searching through *all* the known OPTIMADE databases *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```
Comment (CasperWA) [inline clients.md:27]:
```suggestion
It allows for searching through *all* the known OPTIMADE databases from the [OPTIMADE providers list](https://providers.optimade.org) *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```

**Review by CasperWA — CHANGES_REQUESTED** [review]
I didn't want to put all here initially since anyone may create and host an OPTIMADE API implementation - hence this correction/clarification.

File: clients.md (hunk 1 above)
Comment (CasperWA) [reply clients.md:27]: And even then it's a bit untrue, since there are some listed in the providers list which are not available on optimade.science.

Comment (blokhin) [reply clients.md:27]: OK, we can write _all the official_, no problem.

Concerning the possible mismatches: this is due to caching. The *optimade.science* isn't updated as frequently as the source on GitHub. Now the cache is synced with each new release of the [isomorphic client](https://github.com/tilde-lab/optimade-client). Although suboptimal, this was an easiest option, subject to change, of course. In fact, retrieving the providers list must be very quick, and we cannot do that on-the-fly in the browser: the recursive lookup takes about one minute. No one will wait for a minute during the page load. So here's why 😊

**Review by gmrigna — APPROVED** [review]

**Comment by blokhin** [comment]
@rartino @ml-evs please approve

**Review by ml-evs — APPROVED** [review]
Hi @blokhin, sorry for missing the notification a couple of days ago and thanks for adding Casper's suggestions.

File: clients.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The filtering can be done either using the OPTIMADE filter language (see the spe
 
 ## optimade.science
 
-An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Evgeny Blokhin](https://tilde.pro){:target="_blank"} utilizing [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available.
-It allows for searching through *multiple* OPTIMADE databases using the OPTIMADE filter language (see the specification for more information) and the power of CORS.
+An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file. Can be run from the server, USB-stick, etc.
```
Comment (CasperWA) [inline clients.md:26]:
```suggestion
An [open-source](https://github.com/tilde-lab/optimade.science){:target="_blank"} web client developed by [Tilde MI](https://tilde.pro){:target="_blank"} utilizing the [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS){:target="_blank"} is available as a single universal `index.html` file.
Can be run from the server, USB-stick, etc.
```

File: clients.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ The filtering can be done either using the OPTIMADE filter language (see the spe
...
+It allows for searching through *all* the known OPTIMADE databases from the [OPTIMADE providers list](https://providers.optimade.org) *simultaneously* using the OPTIMADE filter language (see the specification for more information).
```
Comment (CasperWA) [inline clients.md:27]: While this is *obviously* grating a bit with these *obvious* highlights and comparisons, I think it's fair to highlight the current differences between this and other clients for sure - just wanted to comment on this - no further clarifications or changes are requested from my side.

**Review by CasperWA — CHANGES_REQUESTED** [review]
Only a very minor change - we use the philisophy "a single line per single sentence" to have clearer `git diff`s - or at least to make it less opaque when changes to the text are made.

After this it's good to go from my side if there are no other extra changes.

**Review by ml-evs — APPROVED** [review]
I'd like to make another PR to the site, so just going to merge this one as it looks ready.

**Pull request merged**